hack/reference-docs-gen-config.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
would you mind adding a readme in /hack that explains a bit what these files you're adding are for, and link to any reference docs?